apps/movie-quotes-api/quotes/quotes.cjs

```diff
@@ -0,0 +1,23 @@
+'use strict'
```

Comment:
why was this client generated in this folder?

Reply:
Oh, that's a leftover — removing...